num2/num2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите координату x координата центра квадрата");
+        Console.WriteLine("-------------------------");
+        double x1 = Convert.ToDouble(Console.ReadLine());
```

Comment:
Методы класса `Convert` лучше не использовать без особых причин (считаются устаревшими). Лучше воспользоваться методом `Parse`:
```suggestion
        double x1 =double.Parse(Console.ReadLine());
```